<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(219, 224, 208);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="menu_widget_6" native="true">
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	
	background-color: rgb(185, 171, 154);
width:200px;
border-radius:15px;
}

QPushButton{
font: 63 12pt &quot;Bahnschrift SemiBold&quot;;
border:none;
text-align:left;
color: rgb(101, 107, 117);
height:33px;
padding-left:10px;

}
QPushButton:hover{
border-radius:5px;

	
	background-color: rgb(236, 218, 197);
}
QPushButton:pressed{
	
	color: rgb(0, 0, 0);
	border:2px solid;
	border-color: rgb(0, 0, 0);


}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QPushButton" name="inventaire_btn">
           <property name="text">
            <string>Inventaire</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/tirelire.png</normaloff>../IMAGES/icones/tirelire.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="achats_btn">
           <property name="text">
            <string>Achats</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/panier.png</normaloff>../IMAGES/icones/panier.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stock_btn">
           <property name="text">
            <string>Stock</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/boutique.png</normaloff>../IMAGES/icones/boutique.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="client_btn">
           <property name="text">
            <string>Clients</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/utilisateurs-alt.png</normaloff>../IMAGES/icones/utilisateurs-alt.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="employee_btn">
           <property name="text">
            <string>Employés</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/mallette.png</normaloff>../IMAGES/icones/mallette.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fournisseurs_btn">
           <property name="text">
            <string>Fournisseurs</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/cote-camion.png</normaloff>../IMAGES/icones/cote-camion.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="referentiel_btn">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
border-radius:7.5px;
background-color: rgb(172, 126, 116);</string>
           </property>
           <property name="text">
            <string>Référentiel</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../IMAGES/icones/livre-alt (1).png</normaloff>../IMAGES/icones/livre-alt (1).png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="user_btn">
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../IMAGES/icones/ajout-dutilisateur.png</normaloff>../IMAGES/icones/ajout-dutilisateur.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{

background-color: rgb(255, 255, 255);
border-radius:15px;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:2px solid;
border-radius:16px;
width:38px;
height:28.4px;

}
QPushButton:hover{
border-color: rgb(163, 221, 61);

}
QPushButton:pressed{
	background-color: rgb(163, 221, 61);

}

QLineEdit{
border:2px solid;
height:40px;
border-radius:15px;
color: rgb(101, 107, 117);
font: 63 10pt &quot;Bahnschrift SemiBold&quot;;
padding-left:10px;

}
QLineEdit:hover{
	border-color: rgb(163, 221, 61);


}
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>13</number>
          </property>
          <property name="bottomMargin">
           <number>13</number>
          </property>
          <item>
           <widget class="QPushButton" name="menu_btn_6">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border:none;}
QPushButton:hover{
	background-color: rgb(189, 191, 228);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../IMAGES/icones/barres-decalees.png</normaloff>../IMAGES/icones/barres-decalees.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>309</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="styleSheet">
                <string notr="true">QTabWidget{

font: 63 12pt &quot;Bahnschrift &quot;;
border:1px solid;
border-radius:0px;

}</string>
               </property>
               <property name="tabPosition">
                <enum>QTabWidget::North</enum>
               </property>
               <property name="tabShape">
                <enum>QTabWidget::Rounded</enum>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Ref</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QTreeWidget" name="treeWidget">
                   <property name="styleSheet">
                    <string notr="true">QTreeWidget{
border:1px solid;
border-radius:0px;

}</string>
                   </property>
                   <column>
                    <property name="text">
                     <string>Produits</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Transports</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Machines</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Equipements</string>
                    </property>
                   </column>
                   <item>
                    <property name="text">
                     <string> Olives vertes</string>
                    </property>
                    <property name="text">
                     <string>CamionS12</string>
                    </property>
                    <property name="text">
                     <string>U23</string>
                    </property>
                    <property name="text">
                     <string>U</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Olives noires</string>
                    </property>
                    <property name="text">
                     <string>CamionS13</string>
                    </property>
                    <property name="text">
                     <string>U24</string>
                    </property>
                    <property name="text">
                     <string>U</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Huile </string>
                    </property>
                    <property name="text">
                     <string>Citerne67</string>
                    </property>
                    <property name="text">
                     <string>CO</string>
                    </property>
                    <property name="text">
                     <string>C</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Ahevous</string>
                    </property>
                    <property name="text">
                     <string>CamionS13</string>
                    </property>
                    <property name="text">
                     <string>A12</string>
                    </property>
                    <property name="text">
                     <string>A</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Stats</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QStackedWidget" name="stackedWidget">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="page_3">
                    <layout class="QGridLayout" name="gridLayout_4">
                     <item row="0" column="0">
                      <layout class="QVBoxLayout" name="verticalLayout_5">
                       <item>
                        <widget class="QWidget" name="widget_5" native="true">
                         <property name="styleSheet">
                          <string notr="true"/>
                         </property>
                         <layout class="QHBoxLayout" name="horizontalLayout_7">
                          <item>
                           <widget class="QLabel" name="label_2">
                            <property name="maximumSize">
                             <size>
                              <width>16777215</width>
                              <height>40</height>
                             </size>
                            </property>
                            <property name="styleSheet">
                             <string notr="true">color: rgb(79, 79, 79);
font: 63 18pt &quot;Bahnschrift SemiBold&quot;;
padding:1px;


</string>
                            </property>
                            <property name="text">
                             <string>Dépenses </string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                             <spacer name="horizontalSpacer_3">
                              <property name="orientation">
                               <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="sizeHint" stdset="0">
                               <size>
                                <width>40</width>
                                <height>20</height>
                               </size>
                              </property>
                             </spacer>
                            </item>
                            <item>
                             <widget class="QPushButton" name="gauche_btn_2">
                              <property name="maximumSize">
                               <size>
                                <width>20</width>
                                <height>20</height>
                               </size>
                              </property>
                              <property name="styleSheet">
                               <string notr="true">QPushButton{
border:none;

}
QPushButton:hover{
	border-color: rgb(192, 194, 232);
border-radius:10px;
border:2px solid;
}
QPushButton:pressed{
	background-color: rgb(192, 194, 232);




}


</string>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                              <property name="icon">
                               <iconset>
                                <normaloff>../IMAGES/icones/angle-petit-gauche (4).png</normaloff>../IMAGES/icones/angle-petit-gauche (4).png</iconset>
                              </property>
                              <property name="iconSize">
                               <size>
                                <width>30</width>
                                <height>30</height>
                               </size>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QPushButton" name="droite_btn_2">
                              <property name="maximumSize">
                               <size>
                                <width>20</width>
                                <height>20</height>
                               </size>
                              </property>
                              <property name="styleSheet">
                               <string notr="true">QPushButton{
border:none;

}
QPushButton:hover{
	border-color: rgb(192, 194, 232);
border-radius:10px;
border:2px solid;
}
QPushButton:pressed{
	background-color: rgb(192, 194, 232);




}



</string>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                              <property name="icon">
                               <iconset>
                                <normaloff>../IMAGES/icones/angle-petit-droit.png</normaloff>../IMAGES/icones/angle-petit-droit.png</iconset>
                              </property>
                              <property name="iconSize">
                               <size>
                                <width>30</width>
                                <height>30</height>
                               </size>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QGraphicsView" name="graphicsView">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>420</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="page_2">
                    <layout class="QGridLayout" name="gridLayout_5">
                     <item row="0" column="0">
                      <layout class="QVBoxLayout" name="verticalLayout_6">
                       <item>
                        <widget class="QWidget" name="widget_7" native="true">
                         <property name="styleSheet">
                          <string notr="true">QPushButton{border:none;}

QPushButton:pressed{
border-radius:10px;
	background-color: rgb(229, 109, 0);}




</string>
                         </property>
                         <layout class="QHBoxLayout" name="horizontalLayout_6">
                          <item>
                           <widget class="QLabel" name="label_3">
                            <property name="maximumSize">
                             <size>
                              <width>16777215</width>
                              <height>40</height>
                             </size>
                            </property>
                            <property name="styleSheet">
                             <string notr="true">color: rgb(79, 79, 79);
font: 63 18pt &quot;Bahnschrift SemiBold&quot;;
padding:1px;


</string>
                            </property>
                            <property name="text">
                             <string>Revenus</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <layout class="QHBoxLayout" name="horizontalLayout_5">
                            <item>
                             <spacer name="horizontalSpacer_2">
                              <property name="orientation">
                               <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="sizeHint" stdset="0">
                               <size>
                                <width>40</width>
                                <height>20</height>
                               </size>
                              </property>
                             </spacer>
                            </item>
                            <item>
                             <widget class="QPushButton" name="gauche_btn">
                              <property name="maximumSize">
                               <size>
                                <width>20</width>
                                <height>20</height>
                               </size>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                              <property name="icon">
                               <iconset>
                                <normaloff>../IMAGES/icones/angle-petit-gauche (4).png</normaloff>../IMAGES/icones/angle-petit-gauche (4).png</iconset>
                              </property>
                              <property name="iconSize">
                               <size>
                                <width>30</width>
                                <height>30</height>
                               </size>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QPushButton" name="droite_btn">
                              <property name="maximumSize">
                               <size>
                                <width>20</width>
                                <height>20</height>
                               </size>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                              <property name="icon">
                               <iconset>
                                <normaloff>../IMAGES/icones/angle-petit-droit.png</normaloff>../IMAGES/icones/angle-petit-droit.png</iconset>
                              </property>
                              <property name="iconSize">
                               <size>
                                <width>30</width>
                                <height>30</height>
                               </size>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QGraphicsView" name="graphicsView_2">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>420</height>
                          </size>
                         </property>
                         <property name="frameShadow">
                          <enum>QFrame::Plain</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionParam_tres">
   <property name="text">
    <string>Paramètres</string>
   </property>
  </action>
  <action name="actionEdition">
   <property name="text">
    <string>Edition</string>
   </property>
  </action>
  <action name="actionSession">
   <property name="text">
    <string>Session</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inventaire_btn</tabstop>
  <tabstop>achats_btn</tabstop>
  <tabstop>stock_btn</tabstop>
  <tabstop>client_btn</tabstop>
  <tabstop>employee_btn</tabstop>
  <tabstop>fournisseurs_btn</tabstop>
  <tabstop>referentiel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
